build/scripts/post-release.psm1

```diff
@@ -70,11 +77,19 @@ $content
 
   See [CHANGELOG](https://github.com/$gitRepository/blob/$tag/src/$projectName/CHANGELOG.md) for details.
 
+
 "@
   }
 
-  if ($version -match '-alpha' -or $version -match '-beta' -or $version -match '-rc')
+  if ($isPrerelease -eq $true)
   {
+    $notes =
+@"
+The following changes are from the previous release [$previousVersion](https://github.com/$gitRepository/releases/tag/$tagPrefix$previousVersion).
```

Comment:
`The following changes are from the previous release` - finding it slightly confusing. do we need this?

Reply:
If you look at a release, say 1.9.0: https://github.com/open-telemetry/opentelemetry-dotnet/releases/tag/core-1.9.0.

What you will see is most packages say: `No notable changes`.

This is because the content is spread out over all the released alpha/beta/rcs:

https://github.com/open-telemetry/opentelemetry-dotnet/releases/tag/core-1.9.0-rc.1
https://github.com/open-telemetry/opentelemetry-dotnet/releases/tag/core-1.9.0-alpha.1

The point of this "The following changes are from the previous release XYZ" text is to tell users they need to dig into the other releases if they really want to find everything that changed in the final drop. I'm open to changing the text if you think something else would be better/more clear but I do feel we should say _something_ otherwise users might think there were no changes.

PS: I edited the actual 1.9.* releases on GH in the main repo to use what this PR is proposing so you can see a more real world example of what it will look like.

Reply:
got it